src/components/map/MapContainer.jsx

```
@@ -1,45 +1,96 @@
+    searchType,
+    startLocation,
+    endLocation,
+    setStartLocation,
+  } = usePlaceStore();
+
+  const showError = (title, icon = 'error') => {
+    Swal.fire({ title, icon });
+  };
+
+  const waitForKakaoMaps = (callback, retries = 10) => {
+    if (window.kakao && window.kakao.maps) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)